### DISCUSSION TITLE
Cannot Power EBAZ4205

### DISCUSSION THREAD

**Discussion by jay20162016** [xjtuecho/EBAZ4205#42] [Troubleshooting]
Hello! I've received the EBAZ4205 board (from aliexpress) with the TF card, JTAG, UART headers already soldered on. I've moved D23 to D24.

However, when I connect it to my computer (using a Xilinx Platform Cable clone and a GPU MOLEX connector to J4), Vivado says "no devices detected". No LEDs, not even the power LED light up. 

Any help would be appreciated. Thanks in advance!

**Comment by nic3-14159** [comment]
Hello! Which way did you solder the D24 diode? If it is backwards, that could cause it to not turn on. Also, I'm not sure by what you mean with the molex connector on J5. Perhaps you could take a picture?

**Reply by jay20162016** [reply]
I (remember) that I soldered the D24 diode the same way as the others, just as marked on the pcb. I connected my computer's PSU's 3x2 pin (gpu) molex connector to J4 (sorry, typo).

**Reply by nic3-14159** [reply]
That should be the correct orientation, and the molex connector should be supplying power. Just making sure, does it power on if you remove the JTAG cable?

**Reply by jay20162016** [reply]
It doesn't power on if I remove the JTAG cable.

**Reply by jay20162016** [reply]
Whops, I accidentally just short circuited my board accidentally while probing the power circuits :( . If I didn't make a mistake, there is power between both ends of the diode to GND. Hopefully I didn't break anything...

**Reply by jay20162016** [reply]
Seems like nothing is broken, and the power supply still works :)

**Comment by jay20162016** [comment]
It works now, no idea why...
